distribution/src/bin/elasticsearch-env

```diff
@@ -45,7 +45,7 @@ else
 fi
 
 if [ ! -x "$JAVA" ]; then
-  echo "could not find java; set JAVA_HOME or ensure java is in PATH"
+  echo "could not find java; set JAVA_HOME or ensure java is in PATH" >&2
```

Comment:
I notice that you changed the mode on this file. Is that intentional? We do not intend for this file to be executed directly, so it doesn't need the execute bit set.

Reply:
I did? I only redirected the err msg to stderr instead of stdout, am I missing something?

Reply:
Thanks, good catch! Truly odd, I definitely didn't do any chmod's yet something changed it. I've reverted it in 1b4f895